src/pages/Main/RequestForHelp.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import  RequestForHelpForm  from '@components/RequestForHelp';
+import options from "@utils/Options";
+import { notification } from 'antd';
+import Api from '@api';
+
+const openNotification = (message, desc) => {
+    notification.open({
+      message: message,
+      description: desc,
+    });
+  };
+
+const RequestForHelp = () => {
+    const initialValues = {region: ["KA", "5"]};
+
+    const handleSubmit = async (formData) => {
```

Comment:
we have to move this to saga. trigger reducer dispatch here, saga takes care of it and posts result in state